demos/imagenet_classification_demo/README.md

```diff
@@ -0,0 +1,111 @@
+# Imagenet Classification C++ Demo
```

Comment:
The focus on ImageNet here seems unwarranted. Is the demo actually tied to ImageNet in any way? It seems that it should work with any network, as long as an appropriate file with labels is supplied.